bower.json

```diff
@@ -53,5 +52,9 @@
     "spice-html5-bower": {
       "main": []
     }
+  },
+  "resolutions": {
```

Comment:
Since we're running bower with `-F`, we *probably* don't need to add the resolutions section, right? Or do we?